readthedocs/core/middleware.py

```diff
@@ -22,17 +14,6 @@ class ReadTheDocsSessionMiddleware(SessionMiddleware):
     An overridden session middleware with a few changes.
 
     - Doesn't create a session on logged out doc views.
-    - Uses a fallback cookie for browsers that reject SameSite=None cookies
-    - Modifies Django's behavior of treating SESSION_COOKIE_SAMESITE=None
-      to mean SameSite unset and instead makes it mean SameSite=None
-
-    This overrides and replaces Django's built-in SessionMiddleware completely.
-    Much of this middleware is duplicated from Django 2.2's SessionMiddleware.
-
-    In Django 3.1, Django will fully support SameSite=None cookies.
-    However, we may still need this middleware to support browsers that reject
-    SameSite=None cookies.
-    https://www.chromium.org/updates/same-site/incompatible-clients
     """
 
     # Don't set a session cookie on these URLs unless the cookie is already set
```

Comment:
We should likely update these URLs to include the Addons API, and other ones. I don't think this is really doing anything anymore?

Reply:
Hmm, true, but also, I don't see any cookies being set in our docs domains. I think this middleware made sense for when we were hitting the API from our main domain. I guess we can delete it now.

Reply:
Opened https://github.com/readthedocs/readthedocs.org/issues/11692